server/service/integration_core_test.go

```diff
@@ -44,6 +44,17 @@ func (s *integrationTestSuite) SetupSuite() {
 	s.withServer.SetupSuite("integrationTestSuite")
 }
 
+/*
+func (s *integrationTestSuite) SetupTest() {
+	t := s.T()
+	ctx := context.Background()
+	appConf, err := s.ds.AppConfig(ctx)
+	require.NoError(t, err)
+	require.NotEmpty(t, appConf.OrgInfo.OrgName)
+	require.NotEmpty(t, appConf.ServerSettings.ServerURL)
+}
+*/
```

Comment:
Should we remove this if it's not needed?

Reply:
Oops, yes indeed! :D